learnersync.go

```diff
@@ -169,11 +169,39 @@ func (s *Sync) Close() {
 
 func (s *Sync) ignorable(path string) bool {
 	for _, pattern := range s.Ignore {
+		// Strip trailing slash from pattern to handle "node_modules/" like "node_modules"
+		pattern = strings.TrimSuffix(pattern, "/")
+
+		// First, try matching against the full path
 		matched, err := filepath.Match(pattern, path)
 		fatalIfSet(err)
 		if matched {
 			return true
 		}
+
+		// For literal patterns (no wildcards), check if they match as path components.
+		// Wildcard patterns are already handled correctly by filepath.Match above.
+		if !strings.Contains(pattern, "*") {
+			// If pattern starts with /, it should only match at the root
+			if strings.HasPrefix(pattern, string(filepath.Separator)) {
```

Comment:
Should this be an explicit:

```suggestion
			if strings.HasPrefix(pattern, string("/")) {
```

Because at this point we're talking about the pattern, rather than the system that the sync is running on. 

I think it's correct to use the Separator when looking at the path, but maybe not when looking at the pattern. Does that sound right?